gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/instrumentation/reaper/ReaperClassLoadTransform.kt

```diff
@@ -103,23 +103,15 @@ class ReaperClassLoadMethodVisitor(
   override fun visitCode() {
```

Comment:
I renamed this file to `ReaperClassLoadTransform` to match `SnapshotsPreviewRuntimeRetentionTransform`, as I was trying to search for the file and `ReaperClassLoad` managed to be tougher to find than I wanted.

Reply:
sg!